lib/services/azuresqldb/client.js

```diff
@@ -0,0 +1,240 @@
+/* jshint camelcase: false */
+/* jshint newcap: false */
+var HttpStatus = require('http-status-codes');
+var request = require('request');
+var async = require('async');
+
+var API_VERSION_FOR_TOKEN = '2015-05-01-preview';
+var API_VERISON_FOR_RESOURCE_GROUP = '2015-11-01';
+var API_VERSION_FOR_SQL = '2014-04-01-preview';
+var AZURE_RESOURCE_MANAGER_ENDPOINT = 'https://management.azure.com/';
+var MOONCAKE_RESOURCE_MANAGER_ENDPOINT = 'https://management.chinacloudapi.cn/';
+var AZURE_AD_ENDPOINT = 'https://login.windows.net/';
+var MOONCAKE_AD_ENDPOINT = 'https://login.chinacloudapi.cn/';
+
+var sqldbOperations = function (log, azure) {
+    this.log = log;
+    this.azure = azure;
+    this.resourceManagerEndpointUrl = (azure.environment == 'AzureChinaCloud') ? MOONCAKE_RESOURCE_MANAGER_ENDPOINT : AZURE_RESOURCE_MANAGER_ENDPOINT;
+    this.activeDirectoryEndpointUrl = (azure.environment == 'AzureChinaCloud') ? MOONCAKE_AD_ENDPOINT : AZURE_AD_ENDPOINT;
```

Comment:
Sorry. I forgot that you didn't have a subscription in Mooncake. I'll test it.
However, I'm not sure whether SQL server is supported in Mooncake.